release_images.yml

```diff
@@ -2,57 +2,28 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "pytorch"
-    version: "1.10.2"
-    arch_type: "x86"
+    framework: "tensorflow"
+    version: "2.9.0"
     customer_type: "e3"
+    arch_type: "x86"
     training:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      cuda_version: "cu113"
-      os_version: "ubuntu20.04"
-      example: False
-      disable_sm_tag: False
-      force_release: False
-    inference:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      cuda_version: "cu113"
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py39"]
       os_version: "ubuntu20.04"
+      cuda_version: "cu112"
       example: False
-      disable_sm_tag: False
+      disable_sm_tag: True
```

Comment:
this should be set to False - there are tags that we use for canaries that mimic the sm tags